upgrade/v14xto150/upgrade.go

```diff
@@ -26,10 +26,6 @@ func UpgradeResources(namespace string, lhClient *lhclientset.Clientset, kubeCli
 		return err
 	}
 
-	if err := upgradeVolumes(namespace, lhClient, resourceMaps); err != nil {
-		return err
-	}
-
 	if err := upgradeWebhookAndRecoveryService(namespace, kubeClient); err != nil {
```

Comment:
Good fix! Should we wait for the old webhook pods to be completely removed first? If not request might be sending to old webhooks and mess up the states

Reply:
Good idea. Should we wait for a period, e.g. up to 300 seconds?

Reply:
300 sounds good. Then if timeout while waiting, we crash the longhorn-manager so that it retries webhook pod deletion. As long as we not move forward with CR upgrade while the old webhook pods exist, we are good.

Reply:
Sounds good, also it's better to have logs.

Reply:
Addressed. Thank you.